queue_services/entity-filer/tests/unit/test_worker/test_worker.py

```diff
@@ -422,66 +422,6 @@ async def test_process_filing_completed(app, session, mocker):
     assert business.last_ar_date
 
 
-async def test_correction_filing(app, session, mocker):
```

Comment:
not valid anymore. CP will not go to PENDING_CORRECTION status

Reply:
That status was for "local corrections", right?

Yes, local corrections are removed from UI code. The only possibility is something like a conversion correction (as above), but even that's not allowed, I think.